src/router/index.ts

```diff
@@ -1,7 +1,9 @@
-import { createRouter, createWebHistory } from '@ionic/vue-router';
-import { RouteRecordRaw } from 'vue-router';
-import Home from '@/views/Home.vue'
-import Login from '@/views/Login.vue'
+import { createRouter, createWebHistory } from '@ionic/vue-router'
+import { RouteRecordRaw } from 'vue-router'
+import Login from "@/views/Login.vue"
```

Comment:
Why some imported files are in single quotes and some have double ?

Reply:
all are in single quote now